lib/std/os/uefi/tables/boot_services.zig

```diff
@@ -144,16 +145,16 @@ pub const BootServices = extern struct {
     uninstallMultipleProtocolInterfaces: *const fn (handle: *Handle, ...) callconv(.C) Status,
```

Comment:
These two are the only exception. The compiler said:
```
$ zig build-exe -target x86_64-uefi x.zig --zig-lib-dir liblib/std/os/uefi/tables/boot_services.zig:142:82: error: variadic function must have 'C' calling convention
    installMultipleProtocolInterfaces: *const fn (handle: *Handle, ...) callconv(cc) Status,
                                                                                 ^~
$ zig build-exe -target x86_64-uefi x.zig --zig-lib-dir lib
lib/std/os/uefi/tables/boot_services.zig:145:84: error: variadic function must have 'C' calling convention
    uninstallMultipleProtocolInterfaces: *const fn (handle: *Handle, ...) callconv(cc) Status,
                                                                                   ^~
```
`git grep "\.\.\." lib/std/os/uefi` also yields only these two functions.

Reply:
This error originates here:

https://github.com/ziglang/zig/blob/2e424e019f6e7e12656a045ed4b9804f786dede9/src/Sema.zig#L8962-L8969

Maybe the condition should be relaxed to include all "C-like" calling conventions?

Reply:
Does C-like mean any non-`.Unspecified` calling convention?

Reply:
Please introduce a function `callConvSupportsVarArgs` and make it true for C, false for everything else. And then you can add more calling conventions to the set if you verify that they do indeed support var args.

Reply:
Ooh I see! Sounds like a good approach to start with.

Reply:
There are TODOs here now.